frappe/public/js/frappe/form/layout.js

```diff
@@ -98,27 +100,40 @@ frappe.ui.form.Layout = class Layout {
 	}
 
 	show_message(html, color, permanent = false) {
-		if (this.message_color) {
-			// remove previous color
-			this.message.removeClass(this.message_color);
+		/**Render a message block with its own color and close button
+		 * @param {String} html - message or HTML to be displayed
+		 * @param {String} color - color of the block
+		 * @param {Boolean} permanent - if true, the block will not have a close button
+		 */
```

Comment:
I think in JS the convention is to have docstrings above the function declaration, not inside.